index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Microverse Portofolio project</title>
+</head>
+
+<body>
+    <header>
+        <a class="logo">Dorcas</a>
+        <nav class="Union">
+            <img src="Union.png">
+        </nav>
+    </header>
+    <main class="content">
+        <section class="subcontent">
+            <h1> I'm Dorcas Uwimbabazi<br>
+                 Glad to see <br>
+                 you! </h1>
+                 
+                 <p>
+                    I’m a software developer! I can help you<br>
+                    build a product , feature or website Look<br>
+                    through some of my work<br>
+                    and experience! If you like what you see and<br>
+                    have a project you need coded, don’t<br>
+                    hestiate to contact me.
+                 </p>
+                 
+                 <div class="social_media">
+                    <h4>LET'S CONNECT</h4>
+                    <ul>
+                        <li>
+                            <a href="#"></a>
+                            <img src="Vector.png" alt="twitter picture" >
+                        </li>
+                    </ul>
+                    <ul>
+                        <li>
+                            <a href="#"></a>
+                            <img src="Vector (2).png" alt="linkedin picture" />
+                            <img src="Vector (3).png" alt="linkedin picture">
+                            
+                        </li>
+                    
+                    </ul>
+
+                    <ul>
+                        <li>
+                            <a href="#"></a>
+                            <img src="Vector (1).png" alt="Gmail picture" />
+                        </li>
+                    </ul>
+
+                    <ul>
+                        <li>
+                            <a href="#"></a>
+                            <img src="Vector (4).png" alt="github picture">
+                        </li>
+                    </ul>
+
+                    <ul>
+                        <li>
+                            <a href="#"></a>
+                            <img src="Vector (5).png" alt=" peace sign picture" >
+                        </li>
```

Comment:
- [ ] Image is not showing on your mobile website. Kindly work on it and fix the error. :+1: